crypto/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "crypto"
+version = "0.1.0"
+authors = ["Tock Project Developers <tock-dev@googlegroups.com>"]
+
+build = "build.rs"
+
+[dependencies]
+rust-libcore = "*"
+cortexm4 = { path = "../arch/cortex-m4" }
+kernel = { path = "../kernel" }
```

Comment:
I don't think you depend on either of these packages. you can remove this whole section.